index.js

```diff
@@ -78,9 +78,12 @@ module.exports = function (log, isReady) {
         log.since.once(function (since) {
```

Comment:
**could be over my head**

these `log.since.once` methods are observeables, meaning the cb inside is called each time there's an update?
Do we need to worry about this setting up hundreds of pull streams if there are hundreds of changes to `sv.since` and `log.since`

Reply:
Isn't that the point of once?